framework/plugin.mk

```diff
@@ -0,0 +1,45 @@
+# make sure MOOSE_DIR is set
```

Comment:
I see that you include this file in the Makefile for the example problem. Is that the only place this is used? Do we not use that for the regression tests? Maybe it's worth putting a comment at the top of this file to explain what it's for.

Reply:
there is a regression test for this